### DISCUSSION TITLE
The remote certificate is invalid according to the validation process.

### DISCUSSION THREAD

**Discussion by GersonCondula** [XiaoFaye/WooCommerce.NET#735] [Q&A]
The underlying connection was closed: A trust relationship for the SSL/TLS secure channel could not be established.

How can i use auth by https and write verifyssl to false?

**Comment by XiaoFaye** [comment]
Check here: https://www.thecodebuzz.com/bypass-ssl-certificate-net-core-windows-linux-ios-net/
